intranet/apps/eighth/models.py

```diff
@@ -499,7 +504,11 @@ def save(self, *args, **kwargs):  # pylint: disable=arguments-differ
         super(EighthBlock, self).save(*args, **kwargs)
 
     def next_blocks(self, quantity=-1):
-        """Get the next blocks in order."""
+        """Get the next blocks in order.
+
+        Returns: a QuerySet with the specified number of blocks after this block. If `quantity` is passed, then the QuerySet will not be
```

Comment:
`Returns` block is improperly formatted and missing return type.  Needs an `Args` block.